_dev/src/components/product-feed/settings/delivery-time-and-rates/create-custom-carrier.vue

```diff
@@ -0,0 +1,317 @@
+                <b-col>
+                  <div>
+                    <b-input-group
+                      :append="budgetCurrencySymbol"
+                      class="ps_gs-carrier__input-number-group"
+                    >
+                      <b-form-input
+                        type="number"
+                        class="ps_gs-carrier__input-number no-arrows"
+                        size="sm"
+                        v-model.number="carrier[carrier.offerChosen].shippingRateAmount"
+                        :state="validateAmountRate(carrier[carrier.offerChosen].shippingRateAmount)"
+                      />
+                    </b-input-group>
+                  </div>
+                </b-col>
+              </b-row>
+              <div v-else>
+                <b-row>
+                  <b-col class="align-self-center">
+                    <div class="mb-1">
+                      <span
+                        class="font-weight-600"
+                      >
+                        {{
+                          $t('productFeedSettings.deliveryTimeAndRates.estimateStep.shippingRate.overAmount')
+                        }}
+                      </span>
+                      <b-button
+                        class="p-0 ml-1"
+                        variant="text"
+                        v-b-tooltip:psxMktgWithGoogleApp
+                        :title="''"
+                      >
+                        <span class="material-icons-round text-primary mb-0 ps_gs-fz-16 w-16">
+                          info_outlined
+                        </span>
+                      </b-button>
+                    </div>
+                  </b-col>
+                  <b-col class="col-auto mb-2">
+                    <b-input-group
+                      :append="budgetCurrencySymbol"
+                      class="ps_gs-carrier__input-number-group"
+                    >
+                      <b-form-input
+                        type="number"
+                        class="ps_gs-carrier__input-number no-arrows ps_gs-mw-90"
+                        size="sm"
+                        v-model.number="carrier[carrier.offerChosen].shippingRateAmount"
+                        :state="validateAmountRate(carrier[carrier.offerChosen].shippingRateAmount)"
+                      />
+                    </b-input-group>
+                  </b-col>
+                </b-row>
+                <b-row>
+                  <b-col class="align-self-center">
+                    <div class="mb-1">
+                      <span
+                        class="font-weight-600"
+                      >
+                        {{ $t('productFeedSettings.deliveryTimeAndRates.estimateStep.shippingRate.rate') }}
+                      </span>
+                      <b-button
+                        class="p-0 ml-1"
+                        variant="text"
+                        v-b-tooltip:psxMktgWithGoogleApp
+                        :title="''"
```

Comment:
What is this line about?

Reply:
Is for the tooltip text but I don't have it